topics/crystal/index.md

```diff
@@ -0,0 +1,12 @@
+---
+aliases: crystal-lang
+created_by: Ary Borenszweig
```

Comment:
Looking at Crystal's Wikipedia page, should `Juan Wajnerman` also be listed as a creator?

Reply:
Done! 👍